FEATURES.md

```
@@ -230,6 +230,60 @@ These can also be created manually by wrapping an image inside a link, optionall
 [Example Title ![](./link-preview.png)](https://example.com)
 ```
 
+### Snitips
+
+![](./assets/snitip.png)
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)